ISLE/dummy_controller_map.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#define ISLE_BUTTON_SPACE 0
+#define ISLE_BUTTON_ESCAPE 0
```

Comment:
Rather than defining "dummy" values (which I assume won't fulfill any functionality), let's ifdef the entire functionality that depends on it

Reply:
it is possible, but I thought later we could fill this with default values(maybe same as xbox mapping for xinput..?)

Reply:
Either we do it right away or we don't, but I think a stop gap of having dummy values is more confusing than anything